### PR TITLE
[PIL-448, PIL-450] Feature/btc fiat exchange

### PR DESCRIPTION
**Pull request by alazarevski**
Enabled Fiat Exchange to BTC if the user has activated the BTC address
Fixed a bug showing all supported assets in the update form options function

### CODE REVIEW

File: src/screens/Exchange/Exchange.js [hunk 1]
```diff
@@ -979,6 +995,32 @@ class ExchangeScreen extends React.Component<Props, State> {
...
 
+  generateBTCAssetOption = () => {
+    const symbol = 'BTC';
+    const {
+      btcAddresses,
+      btcBalances,
+      baseFiatCurrency,
+      rates,
+    } = this.props;
+    const { address } = btcAddresses[0];
```
Comment (poocart) [inline src/screens/Exchange/Exchange.js]:
```suggestion
    const [{ address }] = btcAddresses;
```

File: src/screens/Exchange/Exchange.js [hunk 2]
```diff
@@ -979,6 +995,32 @@ class ExchangeScreen extends React.Component<Props, State> {
...
+    const assetBalance = rawAssetBalance ? formatAmount(rawAssetBalance) : null;
+    const formattedBalanceInFiat = getFormattedBalanceInFiat(baseFiatCurrency, assetBalance, rates, symbol);
+    const btcAsset = initialAssets.find(e => e.symbol === symbol);
+    return {
+      key: symbol,
+      value: symbol,
+      icon: btcAsset ? btcAsset.iconUrl : '',
+      iconUrl: btcAsset ? btcAsset.iconUrl : '',
+      symbol,
+      ...btcAsset,
```
Comment (poocart) [inline src/screens/Exchange/Exchange.js]: What if no `btcAsset`?

File: src/screens/Exchange/Exchange.js [hunk 3]
```diff
@@ -979,6 +995,32 @@ class ExchangeScreen extends React.Component<Props, State> {
...
+    const addressBalance = btcBalances[address];
+    const rawAssetBalance = addressBalance ? satoshisToBtc(addressBalance.confirmed) : 0;
+    const assetBalance = rawAssetBalance ? formatAmount(rawAssetBalance) : null;
+    const formattedBalanceInFiat = getFormattedBalanceInFiat(baseFiatCurrency, assetBalance, rates, symbol);
+    const btcAsset = initialAssets.find(e => e.symbol === symbol);
+    return {
+      key: symbol,
+      value: symbol,
+      icon: btcAsset ? btcAsset.iconUrl : '',
+      iconUrl: btcAsset ? btcAsset.iconUrl : '',
```
Comment (poocart) [inline src/screens/Exchange/Exchange.js]: `btcAsset` is always present in `initialAssets` so we can just deconstruct `iconUrl` in above and avoid code duplicate.

Comment (alazarevski) [reply src/screens/Exchange/Exchange.js]: flow acted up on that case since find can return undefined

**Pull request merged**